### ISSUE TITLE
 bandwidth usage 

### ISSUE THREAD

**Issue by hamzash9500** [netdata/netdata#9929]
##### Question summary
Question:
Can netdata calculate the bandwidth usage for each process running on the server?

**Comment by ilyam8** [comment]
Hi @hamzash9500 👋 

If you have ebpf installed check `Applications->ebpf net`

**Comment by ilyam8** [comment]
Netdata installer installs it by default. How did you install netdata? Did you check installer logs?

**Comment by hamzash9500** [comment]
i have installed it  by apt repo (apt get install netdata)
os : Ubuntu 18.04.4 LTS
netdata version: v1.23.0

**Comment by ilyam8** [comment]
@thiagoftsm you know better, ping!

**Comment by thiagoftsm** [comment]
Hi @hamzash9500 ,

Please, can you verify that you have the file `/usr/libexec/netdata/plugins.d/ebpf.plugin` with your installation? I am not sure if the deb package had `ebpf.plugin` with Netdata 1.23.0, for the newest versions our SRE team worked to bring it.

Best regards!

**Comment by hamzash9500** [comment]
thank you @ilyam8  
@thiagoftsm  yes I found it and it works as expected, but I have two questions,
 
first one :
can we enable ebpf plugin when i install netdata by binary file? (using wget https://github.com/netdata/netdata/releases/download/v1.23.0/netdata-v1.23.0.gz.run **then** sh netdata-v1.23.0.gz.run )

second question :
how i can store metrics for the last  24 hour in CSV file i used the following URL but it gives data for just the last 10 minutes ago
http://localhost:19999/api/v1/data?chart=apps.bandwidth_sent&format=csv

**Comment by thiagoftsm** [comment]
> can we enable ebpf plugin when i install netdata by binary file? (using wget https://github.com/netdata/netdata/releases/download/v1.23.0/netdata-v1.23.0.gz.run then sh netdata-v1.23.0.gz.run )

Unfortunately this is not possible, I installed now this binary on a VM and the `ebpf.plugin` was not available with it. I also observed that the `ebpf.plugin` is not installed with the latest `netdata-v1.25.0.gz.run`. @prologic and @Ferroin I imagined this was already fixed, are we having problems to build the static binary with ebpf?

> how i can store metrics for the last 24 hour in CSV file i used the following URL but it gives data for just the last 10 minutes ago http://localhost:19999/api/v1/data?chart=apps.bandwidth_sent&format=csv

The API has an additional variable called `after`, by default it has the value `after=-600` that gives 10 minutes of data, if you change your query for `http://localhost:19999/api/v1/data?chart=apps.bandwidth_sent&format=csv&after=-86400`, you will receive the last 24 hours if it is stored inside your database.

Best regards!
